docs/glossary.md

```diff
@@ -0,0 +1,65 @@
+# Glossary
+
+DocArray's scope is at the edge of different fields, from AI to web apps. To make it easier to understand, we have created a glossary of terms used in the documentation. 
+
+## Concept
+
+### `Multimodal Data`
```

Comment:
Stick to "multimodal" (what we use on jina.ai), not "multi modal" or "multi-modal" (or any other variation)

Reply:
sure thx